maintenance_equipment_scrap/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Maintenance Equipments Scrap',
+    'summary': 'Enhance the functionality for Scrapping Equipments',
+    'author': 'Onestein',
+    'website': 'http://www.onestein.eu',
+    'category': 'Human Resources',
```

Comment:
I picked the same category of the Odoo base module 'maintenance', just for keeping consistency with it, but it's fine for me to change it. What do you think?